content/en/blog/_posts/2018-12-04-kubeadm-ga-release.md

```diff
@@ -23,7 +23,7 @@ To keep kubeadm lean, focused, and vendor/infrastructure agnostic, the following
 - Non-critical add-ons, e.g. for monitoring, logging, and visualization
 - Specific cloud provider integrations
 
-Infrastructure provisioning, for example, is left to other SIG Cluster Lifecycle projects, such as the [Cluster API](https://github.com/kubernetes-sigs/cluster-api). Instead, kubeadm covers only the common denominator in every Kubernetes cluster: the [control plane](/docs/concepts/#kubernetes-control-plane). The user may install their preferred networking solution and other add-ons on top of Kubernetes *after* cluster creation.
+Infrastructure provisioning, for example, is left to other SIG Cluster Lifecycle projects, such as the [Cluster API](https://github.com/kubernetes-sigs/cluster-api). Instead, kubeadm covers only the common denominator in every Kubernetes cluster: the {{< glossary_tooltip text="control plane" term_id="control-plane" >}}. The user may install their preferred networking solution and other add-ons on top of Kubernetes *after* cluster creation.
```

Comment:
One note: I think blog posts typically link to documentation pages instead of showing a tooltip.

Reply:
Hi @kbhawkey. Sorry for replying late.
Agree with you that tooltips may break the consistency of link styles on blog pages. How do you think if this `control plane` points to `/docs/concepts/overview/components/#control-plane-components` ?
Personally, I'd like to use the glossary page `/docs/reference/glossary/?all=true#term-control-plane` directly instead of as a tooltip, because it describes the term `control plane` straight.
Or do you have recommended links?

Reply:
Hi @blueabysm.
I think the concept page offers more information to the reader.

Reply:
Thank you @kbhawkey. Two blog pages are updated.